src/frontend/components/property-type/array/edit.tsx

```diff
@@ -13,29 +14,50 @@ type EditProps = Required<EditPropertyPropsInArray>
 
 type ItemRendererProps = {
   onDelete: (event: MouseEvent, property: PropertyJSON) => boolean;
+  index: number;
 }
 
 const ItemRenderer: React.FC<EditProps & ItemRendererProps> = (props) => {
-  const { ItemComponent, property, onDelete } = props
+  const { ItemComponent, property, onDelete, index, record } = props
+  
+  const uniqueDraggableId = btoa(`${JSON.stringify(flat.get(record.params, property.path))}-${property.path}`)
```

Comment:
A unique draggable id has to be constructed to avoid flickering. This is a base64 representation of record's value and it's property path.